src/components/FileDetail/FileDetail.js

```diff
@@ -32,131 +40,155 @@ const FileDetail = ({fileNode, history, match}) => {
+                    data-testid="back-to-filelist"
+                  >
+                    <Button
+                      basic
+                      labeled
+                      icon
+                      size="mini"
+                      labelPosition="left"
+                      floated="left"
+                    >
+                      <Icon name="arrow left" />
+                      Back to All Documents
+                    </Button>
+                  </Link>
+                </Grid.Column>
+              </Grid.Row>
+              <Grid.Row>
+                <Grid.Column>
+                  <Header>
+                    {fileNode.name}
+                    <Button.Group icon labeled size="mini" floated="right">
+                      <Button
+                        icon="pencil"
+                        onClick={() => setDialog('annotation')}
+                        data-testid="edit-button"
+                        content="EDIT"
+                      />
+                      <Button
+                        icon="download"
+                        onClick={e =>
+                          downloadFile(
+                            studyId,
+                            fileNode.kfId,
+                            null,
+                            downloadFileMutation,
+                          )
+                        }
+                        content="DOWNLOAD"
+                      />
+                      <Button
+                        negative
+                        icon="trash alternate"
+                        onClick={e => {
+                          deleteFile({variables: {kfId: fileNode.kfId}});
+                          history.goBack();
+                        }}
+                        content="DELETE"
+                      />
+                    </Button.Group>
+                  </Header>
+                </Grid.Column>
+              </Grid.Row>
+              <Grid.Row stretched>
+                <Grid.Column mobile={8} tablet={8} computer={4}>
+                  <Statistic size="mini">
+                    <Statistic.Label style={{textAlign: 'left'}}>
+                      Status
+                    </Statistic.Label>
+                    <Statistic.Value style={{textAlign: 'left'}}>
+                      <Badge state={sortedVersions[0].node.state} />
+                    </Statistic.Value>
+                  </Statistic>
+                </Grid.Column>
+                <Grid.Column mobile={8} tablet={8} computer={4}>
+                  <Statistic size="mini">
+                    <Statistic.Label style={{textAlign: 'left'}}>
+                      Document Type
+                    </Statistic.Label>
+                    <Statistic.Value style={{textAlign: 'left'}}>
+                      <Label icon basic size="tiny">
+                        <Icon
+                          name={`${fileTypeDetail[fileNode.fileType].icon ||
```

Comment:
I would say conditionally render this to just not show an icon if there's not one set instead of defaulting to the the `question`